### PR TITLE
[ptg.nan-latn-tw.taigipoj] Initial version

### PR DESCRIPTION
**Pull request by shiami**
PhahTaigi POJ Keyman Keyboard Lexical Model for [taigi_poj](https://keyman.com/keyboards/taigi_poj).

As a result of Keyman Developer Workshop.

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

File: experimental/ptg/ptg.nan-latn-tw.taigipoj/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+© 2020 Ngô͘ Ka-bêng
+
+Version 1.0
+
+Description
+-----------
+
+PhahTaigi POJ Keyman Keyboard Lexical Model.
+
+For keyboard you need to use with, please install [PhahTaigi POJ Keyman Keyboard](https://keyman.com/keyboards/taigipoj).
```
Comment (darcywong00) [inline experimental/ptg/ptg.nan-latn-tw.taigipoj/README.md]: The keyboard link should be https://keyman.com/keyboards/taigi_poj

File: experimental/ptg/ptg.nan-latn-tw.taigipoj/source/welcome.htm [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+<h1>Start Using PhahTaigi POJ Keyman Keyboard Lexical Model</h1>
+
+<p>
+    PhahTaigi POJ Keyman Keyboard Lexical Model 1.0
+</p>
+
+<h1>Wordlist Model Documentation</h1>
+
+<p>For keyboard you need to use with, please install the <a href="https://keyman.com/keyboards/taigipoj"?>PhahTaigi POJ Keyman Keyboard</a>.</p>
```
Comment (darcywong00) [inline experimental/ptg/ptg.nan-latn-tw.taigipoj/source/welcome.htm]: Keyboard link should be https://keyman.com/keyboards/taigi_poj

**Comment by shiami** [comment]
@darcywong00 Thank you. Fixed.

File: experimental/ptg/ptg.nan-latn-tw.taigipoj/source/ptg.nan-latn-tw.taigipoj.model.kps [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    <ReadMeFile>readme.htm</ReadMeFile>
+    <MSIFileName></MSIFileName>
+    <MSIOptions></MSIOptions>
+  </Options>
+  <StartMenu>
+    <Folder></Folder>
+    <Items/>
+  </StartMenu>
+  <Info>
+    <Name URL="">PhahTaigi POJ Keyman Keyboard Lexical Model</Name>
```
Comment (darcywong00) [inline experimental/ptg/ptg.nan-latn-tw.taigipoj/source/ptg.nan-latn-tw.taigipoj.model.kps]: The lexical models are separate from Keyman keyboards.

In Keyman Developer, Package Editor, I suggest you change all the name references from `PhahTaigi POJ Keyman Keyboard Lexical Model` to `PhahTaigi POJ Lexical Model`

File: experimental/ptg/ptg.nan-latn-tw.taigipoj/ptg.nan-latn-tw.taigipoj.model_info [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+    "license": "mit",
+    "languages": [
+        "nan-Latn-TW"
+    ],
+	  "description": "PhahTaigi POJ Keyman Keyboard Lexical Model"
```
Comment (darcywong00) [inline experimental/ptg/ptg.nan-latn-tw.taigipoj/ptg.nan-latn-tw.taigipoj.model_info]: I would also change the description to `"PhahTaigi POJ Lexical Model"`

**Comment by shiami** [comment]
@darcywong00 Thanks, done.

**Comment by darcywong00** [comment]
There's still a few more files to fix the lexical model name:
HISTORY.md
README.md
ptg.nan-latn-tw.taigipoj.model_info

The PR should be ready to merge after that

**Comment by darcywong00** [comment]
I'll go ahead and merge this and update these files on a separate PR
> There's still a few more files to fix the lexical model name:
HISTORY.md
README.md
ptg.nan-latn-tw.taigipoj.model_info

**Pull request merged**